sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.netstandard2.0.cs

```diff
@@ -191,4 +191,20 @@ public partial class ProtocolClientOptions : Azure.Core.ClientOptions
     {
         public ProtocolClientOptions() { }
     }
+    public sealed partial class ResponseError
+    {
+        internal ResponseError() { }
+        public string? Code { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<Azure.Core.ResponseError> Details { get { throw null; } }
```

Comment:
it's a bit strange that details of X is a collection of X.

Reply:
And it's even stranger that InnerError is a different type...   but https://github.com/microsoft/api-uidelines/blob/vNext/Guidelines.md#7102-error-condition-responses